src/Platform.Handlers/src/Xamarin.Platform.Handlers/Core/IText.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Xamarin.Platform
 {
-	public interface IText : IFont, ITextAlignment
+	public interface IText : IFont
```

Comment:
Might be worth removing the `TextTransform` as it is limited and might be not sufficient as it is. @hartez mentions a possible case for a `Func<string, string>` being better.